recipes/task.rb

```diff
@@ -45,4 +45,5 @@ class ::Chef::Recipe
   log_directory node['chef_client']['log_dir']
   chef_binary_path node['chef_client']['bin']
   daemon_options node['chef_client']['daemon_options']
+  not_if { ::File.exist?("C:\\Windows\\System32\\Tasks\\#{node['chef_client']['task']['task_name']}") }
```

Comment:
As @tas50 indicated, this would prevent the task from being updated.

Reply:
Yes @btm, I've mentioned it too in my above comment. To make the resource `chef_client_scheduled_task` idempotent I was needed to add this check.

Reply:
@tas50 @btm, This cookbook calculates the `start_time` for frequency `:minute` which seems inappropriate as per `windows_task` resource description. `start_time` needs to be mentioned only for `:once` 
```
frequency

    Ruby Type: Symbol | Default Value: :hourly

    Frequency with which to run the task. Values include: :minute, :hourly, :daily, :weekly, :monthly, :once, :on_logon, :onstart, :on_idle. The :once value requires the start_time property.
``` 
Please check here https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_windows_task.rst#properties

I am trying to modify the recipe like gist shared here https://gist.github.com/dheerajd-msys/e771890bc8bf0f515582c77bba84c7ae.

Please suggest your views for modifications.